### ISSUE TITLE
change name of the repository

### ISSUE THREAD

**Issue by aoyawale** [RedHatSatellite/satellite-ansible-compilation#10]
we should change - to _ so its compatible and easier to do with galaxy/collections in the future. We ran into this with the redhat-cop and repositories there. There's work arounds but someone in the galaxy side has to do it. We can change it now and avoid headaches down the road.

**Comment by ehelms** [comment]
Given the current intent of this repository, I do not believe we want to turn this into a galaxy collection. So I would like to avoid any changes that necessarily aim that direction. I think our goal is to assess what ends up here, and build out new, logical collections. For example, one dedicated to operations or installation.

**Comment by bherrin3** [comment]
@ehelms @jlozadad @evgeni

tl;dr - I believe that we should have this as an Ansible Galaxy and therefore should be renamed to something Ansible Galaxy can consume.

The main reason is that contributors should be able to move content out of their `siloed` repo into this collection and then rebase their repo on this collection's content. 

While some items here might need to be refactored to go into a more `official` collections like `operations`, `installation` or `FAM`, we should be encouraging people to contribute their content to a shared location.

I would like to go ahead with the suggested project name update and any other additions / updates we need to have an uploaded Ansible Galaxy Collection.

**Comment by evgeni** [comment]
ansible-galaxy (the cli tool) supports installing collections from git urls, would that suffice (if we provide the needed metadata in the repo)?

I'd really like to avoid uploading "unfiltered" content to galaxy (the service).

**Comment by kedark3** [comment]
Thanks Evgeni, before we move forward and close this issue, I'd like to ask @wilson-walrus @tonykay @jlozadad @cojmckee @greg-hellings for their thoughts too.

**Comment by bherrin3** [comment]
@evgeni what this doesn't address is the fact that there is rampant duplication of this effort. Just today I ran into another team who had ~~partially~~ completely hand-written a satellite installation repo for VMWare.

Agreed it will be unfiltered, unsupported content, but with proper disclaimer, it should be on Ansible Galaxy. Else, no one can know what they need hasn't already been done.

**Comment by evgeni** [comment]
As a user, I wouldn't look further into something if it says unsupported on the tin. 🤷‍♀️

**Comment by kedark3** [comment]
@evgeni I beg to defer, I have seen many such things that don't say if they are or aren't supported, from Galaxy, but has served as a good starting point, or in somecases like ~90% of what I needed to do what I wanted.

**Comment by greg-hellings** [comment]
Probably no need to disagree with what @evgeni is telling us he would do in this scenario. 🙂

As for changing the name, I don't mind it. But it's also not necessary. When building a collection, unlike roles, they only get imported to Galaxy when a distinct build/publish step is invoked. That step creates a tarball whose name is controlled by the contents of `galaxy.yml`. It's very common to have the name of the collection not match the name of the repository. My repost named ansible_collection_[somenane], but the collection name in my galaxy file dictates the name of the actual collection and build result tar file.

**Comment by ehelms** [comment]
This is my proposal on how we move forward to address support concerns and address the needs of the stakeholders in this repository:

 1) Rename this repository to `satellite-community-collection`
 2) Be a collection named `redhatsatellite.community`
 3) Publish the collection on Ansible Galaxy but not Automation Hub
 4) State in the README that the collection is not supported by Satellite and is a Satellite community driven collection
 5) Encourage within that collection, within the README, transitioning functionality to support the Foreman community

**Comment by bherrin3** [comment]
@ehelms 

Satellite-Ansible-Compilation Meeting notes:
- Is this a collection or "more than a collection" (no playbooks or Ansible Tower items?)
- Tower configuration part of this discussion and collections are still evolving
- Adding `ansible` somewhere to the name so it is easy to know this contains `ansible` content -- `satellite-community-ansible-collection`. This makes it easier to know this collection is ansible-based from a glance (can revisit this naming to get it correct - WIP)... better wordsmiths needed here :)
- collection name `satellite.community` if we can get an exception? The name isn't bad as-is

**Comment by ehelms** [comment]
> * Is this a collection or "more than a collection" (no playbooks or Ansible Tower items?)

I think more than given our goals. If there are playbooks or Tower items that are relevant or being created by multiple groups then let's include them.

> * Adding `ansible` somewhere to the name so it is easy to know this contains `ansible` content -- `satellite-community-ansible-collection`. This makes it easier to know this collection is ansible-based from a glance (can revisit this naming to get it correct - WIP)... better wordsmiths needed here :)

Sounds good to me.

> * collection name `satellite.community` if we can get an exception? The name isn't bad as-is

I think there is an advantage to mapping the Github org to the galaxy org. @evgeni thoughts here?
